src/patients/appointments/AppointmentsList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useHistory } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { TextInput, Button, List, ListItem, Container, Row } from '@hospitalrun/components'
+import { RootState } from '../../store'
+import { fetchPatientAppointments } from '../../scheduling/appointments/appointments-slice'
+
+interface Props {
+  patientId: string
+}
+
+const AppointmentsList = (props: Props) => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const { t } = useTranslation()
+
+  const { patientId } = props
+  const { appointments } = useSelector((state: RootState) => state.appointments)
+  const [searchText, setSearchText] = useState<string>('')
+
+  useEffect(() => {
+    dispatch(fetchPatientAppointments(patientId))
+  }, [dispatch, patientId])
+
+  const list = (
+    // inline style added to pick up on newlines for string literal
+    <ul style={{ whiteSpace: 'pre-line' }}>
+      {appointments.map((a) => (
+        <ListItem action key={a.id} onClick={() => history.push(`/appointments/${a.id}`)}>
+          {`${t('scheduling.appointment.location')}: ${a.location}
+          ${t('scheduling.appointment.reason')}: ${a.reason}
+          ${t('scheduling.appointment.type')}: ${a.type}
```

Comment:
location, reason and type aren't required fields when creating an appointment. Could conditionally render, but this would get ugly... unless someone has an idea 😄